test/wasm2c/add.txt

```diff
@@ -681,8 +687,23 @@ DEFINE_TABLE_FILL(externref)
 #define FUNC_TYPE_T(x) static const char x[]
 #endif
 
+#if (__STDC_VERSION__ >= 201112L) || defined(_Static_assert)
+#define wasm_static_assert(X) _Static_assert(X, "assertion failure")
+#else
+#define wasm_static_assert(X) \
+  extern int(*wasm2c_assert(void))[!!sizeof(struct { int x : (X) ? 2 : -1; })];
+#endif
+
 static u32 w2c_test_add_0(w2c_test*, u32, u32);
 
+#ifndef wasm_multi_ii
```

Comment:
Hmm... is the "wasm_multi_ii" avoidable here? (I haven't got to the part where I dig into what this is used for yet. Will look some more in the next pass)

Reply:
Done.